packages/eslint-plugin/src/rules/no-empty-interface.ts

```diff
@@ -61,13 +61,21 @@ export default util.createRule<Options, MessageIds>({
             context.report({
               node: node.id,
               messageId: 'noEmptyWithSuper',
-              fix: fixer =>
-                fixer.replaceText(
+              fix: fixer => {
+                let typeParam = '';
+                if (node.typeParameters) {
+                  typeParam = sourceCode.text.substring(
+                    node.typeParameters.range[0],
+                    node.typeParameters.range[1],
+                  );
```

Comment:
You should be able to use this method to get the source code for the node
```suggestion
                  typeParam = sourceCode.getText(node.typeParameters);
```

Reply:
Very new to the codebase, thanks!!